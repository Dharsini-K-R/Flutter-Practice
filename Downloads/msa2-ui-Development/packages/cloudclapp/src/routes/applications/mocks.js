export const dockerImage = [
  {
    id: "",
    name: "node",
    slug: "node",
    type: "image",
    publisher: {
      id: "docker",
      name: "Docker",
    },
    created_at: "2016-06-17T17:45:46.898676Z",
    updated_at: "2022-02-04T06:12:22.541527Z",
    short_description:
      "Node.js is a JavaScript-based platform for server-side and networking applications.",
    source: "store",
    popularity: 0,
    categories: [
      {
        name: "application_infrastructure",
        label: "Application Infrastructure",
      },
    ],
    operating_systems: [
      {
        name: "linux",
        label: "Linux",
      },
    ],
    architectures: [
      {
        name: "arm64",
        label: "ARM 64",
      },
      {
        name: "s390x",
        label: "IBM Z",
      },
      {
        name: "ppc64le",
        label: "PowerPC 64 LE",
      },
      {
        name: "386",
        label: "386",
      },
      {
        name: "arm",
        label: "ARM",
      },
      {
        name: "amd64",
        label: "x86-64",
      },
    ],
    logo_url: {
      large:
        "https://d1q6f0aelx0por.cloudfront.net/product-logos/library-node-logo.png",
      small:
        "https://d1q6f0aelx0por.cloudfront.net/product-logos/library-node-logo.png",
    },
    certification_status: "not_certified",
    star_count: 11111,
    pull_count: "1B+",
    filter_type: "official",
  },
];

export const VMImage = [
  {
    architecture: "x86_64",
    creationDate: "2022-04-28T21:51:16.000Z",
    deprecationTime: "2024-04-28T21:51:16.000Z",
    description: "Amazon Linux 2 Kernel 5.10 AMI 2.0.20220426.0 x86_64 HVM gp2",
    enaSupport: true,
    hypervisor: "xen",
    imageId: "ami-0022f774911c1d690",
    imageLocation: "amazon/amzn2-ami-kernel-5.10-hvm-2.0.20220426.0-x86_64-gp2",
    imageOwnerAlias: "amazon",
    imageType: "machine",
    isPublic: false,
    name: "amzn2-ami-kernel-5.10-hvm-2.0.20220426.0-x86_64-gp2",
    ownerId: "137112412989",
    platform: "Amazon Linux",
    productCodes: [],
    rootDeviceName: "/dev/xvda",
    rootDeviceType: "ebs",
    virtualizationType: "hvm",
  },
  {
    architecture: "x86_64",
    creationDate: "2022-04-28T21:51:16.000Z",
    deprecationTime: "2024-04-28T21:51:16.000Z",
    description: "Amazon Linux 2 Kernel 5.10 AMI 2.0.20220426.0 x86_64 HVM gp2",
    enaSupport: true,
    hypervisor: "xen",
    imageId: "ami-0022f774911c1d690",
    imageLocation: "amazon/amzn2-ami-kernel-5.10-hvm-2.0.20220426.0-x86_64-gp2",
    imageOwnerAlias: "amazon",
    imageType: "machine",
    isPublic: false,
    name: "amzn2-ami-kernel-5.10-hvm-2.0.20220426.0-x86_64-gp2",
    ownerId: "137112412989",
    platform: "Amazon Linux",
    productCodes: [],
    rootDeviceName: "/dev/xvda",
    rootDeviceType: "ebs",
    virtualizationType: "hvm",
  },
  {
    architecture: "x86_64",
    creationDate: "2022-04-28T21:51:16.000Z",
    deprecationTime: "2024-04-28T21:51:16.000Z",
    description: "Amazon Linux 2 Kernel 5.10 AMI 2.0.20220426.0 x86_64 HVM gp2",
    enaSupport: true,
    hypervisor: "xen",
    imageId: "ami-0022f774911c1d690",
    imageLocation: "amazon/amzn2-ami-kernel-5.10-hvm-2.0.20220426.0-x86_64-gp2",
    imageOwnerAlias: "amazon",
    imageType: "machine",
    isPublic: false,
    name: "amzn2-ami-kernel-5.10-hvm-2.0.20220426.0-x86_64-gp2",
    ownerId: "137112412989",
    platform: "Amazon Linux",
    productCodes: [],
    rootDeviceName: "/dev/xvda",
    rootDeviceType: "ebs",
    virtualizationType: "hvm",
  },
  {
    architecture: "x86_64",
    creationDate: "2022-04-28T21:51:16.000Z",
    deprecationTime: "2024-04-28T21:51:16.000Z",
    description: "Amazon Linux 2 Kernel 5.10 AMI 2.0.20220426.0 x86_64 HVM gp2",
    enaSupport: true,
    hypervisor: "xen",
    imageId: "ami-0022f774911c1d690",
    imageLocation: "amazon/amzn2-ami-kernel-5.10-hvm-2.0.20220426.0-x86_64-gp2",
    imageOwnerAlias: "amazon",
    imageType: "machine",
    isPublic: false,
    name: "amzn2-ami-kernel-5.10-hvm-2.0.20220426.0-x86_64-gp2",
    ownerId: "137112412989",
    platform: "Amazon Linux",
    productCodes: [],
    rootDeviceName: "/dev/xvda",
    rootDeviceType: "ebs",
    virtualizationType: "hvm",
  },
  {
    architecture: "x86_64",
    creationDate: "2022-04-28T21:51:16.000Z",
    deprecationTime: "2024-04-28T21:51:16.000Z",
    description: "Amazon Linux 2 Kernel 5.10 AMI 2.0.20220426.0 x86_64 HVM gp2",
    enaSupport: true,
    hypervisor: "xen",
    imageId: "ami-0022f774911c1d690",
    imageLocation: "amazon/amzn2-ami-kernel-5.10-hvm-2.0.20220426.0-x86_64-gp2",
    imageOwnerAlias: "amazon",
    imageType: "machine",
    isPublic: false,
    name: "amzn2-ami-kernel-5.10-hvm-2.0.20220426.0-x86_64-gp2",
    ownerId: "137112412989",
    platform: "Amazon Linux",
    productCodes: [],
    rootDeviceName: "/dev/xvda",
    rootDeviceType: "ebs",
    virtualizationType: "hvm",
  },
];
