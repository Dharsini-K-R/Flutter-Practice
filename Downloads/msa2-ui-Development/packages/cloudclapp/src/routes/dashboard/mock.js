export const organisation = [
  { environmentCount: 20, usersCount: 1, prefix: "00p" },
];

export const environments = [
  {
    status: "Critical",
    version: "V1.1.1",
    user: "Bib Dev",
    env: "Development",
    envData:
      "Development Environment for all new projects and major updates. Nginx and PHP added to the Environment",
    deplymentCount: "2",
    cost: "58.2",
  },
];

export const cloudVendors = {
  aws: {
    displayName: "AWS",
    category: "Public Cloud",
    logo: "Process/cloudclapp-wf/CCLA_Adapters/logos/aws.logo",
    credentials_required: ["aws_access_key", "aws_secret_key"],
    services: {
      eks: {
        displayName: "Elastic Container Service for Kubernetes",
        logo: "Process/cloudclapp-wf/CCLA_Adapters/logos/aws_eks.logo",
        workflow: {
          env:
            "Process/cloudclapp-wf/Provision_an_EKS_Cluster__AWS_/Provision_an_EKS_Cluster__AWS_",
          appsDeployment:
            "Process/cloudclapp-wf/Provision_Apps_EKS_Cluster/Provision_Apps_EKS_Cluster",
          scanApp:
            "Process/cloudclapp-wf/Scan_a_App_Container_/Scan_a_App_Container_",
        },
      },
    },
  },
  azure: {
    displayName: "AZURE",
    category: "Public Cloud",
    logo: "Process/cloudclapp-wf/CCLA_Adapters/logos/azure.logo",
    credentials_required: ["azure_client_id", "azure_client_secret"],
    services: {
      aks: {
        displayName: "Azure Kubernetes Service",
        logo: "Process/cloudclapp-wf/CCLA_Adapters/logos/azure_aks.logo",
        workflow: {
          env:
            "Process/cloudclapp-wf/Provision_an_AKS_Cluster__AZURE_/Provision_an_AKS_Cluster__AZURE_",
          appsDeployment:
            "Process/cloudclapp-wf/Provision_an_AKS_Apps__AZURE_/Provision_an_AKS_Apps__AZURE_",
          scanApp:
            "Process/cloudclapp-wf/Scan_a_App_Container_/Scan_a_App_Container_",
        },
      },
    },
  },
  gcp: {
    displayName: "Google Cloud Platform",
    category: "Public Cloud",
    logo: "Process/cloudclapp-wf/CCLA_Adapters/logos/gcp.logo",
    credentials_required: [
      "gcp_service_account_email",
      "gcp_private_key_id",
      "gcp_private_key",
      "gcp_client_id",
    ],
    services: {
      gke: {
        displayName: "Google Kubernetes Engine",
        logo: "Process/cloudclapp-wf/CCLA_Adapters/logos/gcp_gke.logo",
        workflow: {
          env:
            "Process/cloudclapp-wf/Provision_an_GKE_Cluster__GCP_/Provision_an_GKE_Cluster__GCP_",
          appsDeployment:
            "Process/cloudclapp-wf/Provision_Apps_GKE_Cluster/Provision_Apps_GKE_Cluster",
          scanApp:
            "Process/cloudclapp-wf/Scan_a_App_Container_/Scan_a_App_Container_",
        },
      },
    },
  },
};
