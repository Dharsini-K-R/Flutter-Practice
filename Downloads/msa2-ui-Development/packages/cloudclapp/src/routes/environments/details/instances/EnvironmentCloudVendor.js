import React from "react";
import { useTranslation } from "react-i18next";
import { useSelector } from "react-redux";
import { Box, Grid, makeStyles, Typography } from "@material-ui/core";
import PropTypes from "prop-types";
import { getCloudVendors } from "cloudclapp/src/store/designations";
import CloudVendorIcon from "cloudclapp/src/components/cloud-vendor-icon";

const useStyles = makeStyles(({ palette, typography }) => ({
  icon: {
    fontSize: "42px",
    boxSizing: "border-box",
    background: palette.common.white,
    padding: 5,
    borderRadius: 4,
  },
  serviceDetails: {
    fontSize: 13,
    color: typography.body3.color,
    marginLeft: 10,
  },
  serviceName: {
    fontWeight: 600,
  },
  boldText: {
    fontWeight: 500,
  },
}));

const EnvironmentCloudVendor = ({ environment }) => {
  const classes = useStyles();
  const { t } = useTranslation();
  const cloudVendors = useSelector(getCloudVendors);
  const {
    description,
    envIp,
    envHostname,
    cloudVendor: cloudVendorName,
    cloudService: cloudServiceName,
  } = environment;
  const cloudVendor = cloudVendors[cloudVendorName];
  const cloudService = cloudVendor?.services[cloudServiceName];
  return (
    <Grid item>
      <Box display="flex" flexDirection="row" alignItems={"center"}>
        <Box>
          <CloudVendorIcon
            vendor={cloudVendorName}
            service={cloudServiceName}
          />
        </Box>
        <Box p={1}>
          <Typography
            id="INSTANCE_DETAILS_ENVIRONMENT_SERVICE_NAME"
            className={classes.serviceDetails}
          >
            <span className={classes.serviceName}>
              {cloudService?.displayName}
            </span>
            {" - "}
            <span>{cloudVendor?.displayName}</span>
          </Typography>
          <Typography
            id="INSTANCE_DETAILS_ENVIRONMENT_DESCRIPTION"
            className={classes.serviceDetails}
          >
            {description}
          </Typography>
          <Grid container item>
            <Typography
              id="INSTANCE_DETAILS_ENVIRONMENT_IP_ADDRESS"
              className={classes.serviceDetails}
            >
              <span className={classes.boldText}>
                {t("IP Address")}
                {":"}
              </span>{" "}
              {envIp}
            </Typography>
            <Typography
              id="INSTANCE_DETAILS_ENVIRONMENT_HOSTNAME"
              className={classes.serviceDetails}
            >
              <span className={classes.boldText}>
                {t("Host Name")}
                {":"}
              </span>{" "}
              {envHostname}
            </Typography>
          </Grid>
        </Box>
      </Box>
    </Grid>
  );
};

EnvironmentCloudVendor.propTypes = {
  environment: PropTypes.object.isRequired,
  cloudVendor: PropTypes.object.isRequired,
};
export default EnvironmentCloudVendor;
