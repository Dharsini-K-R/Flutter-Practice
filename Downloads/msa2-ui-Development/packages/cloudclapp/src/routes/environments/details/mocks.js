import { DEPLOYMENT_VARIABLES_NAME } from "cloudclapp/src/Constants";
export const deployments = [
  {
    envId: 126,
    deployments: [
      {
        deploymentId: "126027",
        deploymentName: "New APP deployment",
      },
      {
        deploymentId: "126016",
        deploymentName: "New APP deployment 3",
      },
    ],
  },
  {
    envId: 118,
    deployments: [
      {
        deploymentId: "126029",
        deploymentName: "New APP deployment 2",
      },
    ],
  },
];

export const environment = {
  envName: "hto env 24",
  envId: 126,
  envUbiqubeId: "005A126",
  description: "aslkd adsg aslgd ll",
  cloudVendor: "aws",
  cloudService: "eks",
  orgName: "hto Org extRef",
  envIp: "127.0.0.1",
  envHostname: "temp-host",
  username: "a",
  envReference: "88faa4e6-be22-444f-a085-e362273d76af",
  importFlag: false,
  serviceId: null,
  serviceName: "Env",
  envWFUri:
    "Process/cloudclapp-wf/Provision_an_EKS_Cluster__AWS_/Provision_an_EKS_Cluster__AWS_",
  appWFUri:
    "Process/cloudclapp-wf/Provision_an_EKS_Apps__AWS_/Provision_an_EKS_Apps__AWS_",
  deploymentCount: 1,
  userCount: 1,
  scanData: null,
  cclapOwner: 105,
  envEntityId: 306,
  status: "Completed",
};

export const applications = [
  {
    [DEPLOYMENT_VARIABLES_NAME.APPLICATION_NAME]: "NATS0",
    [DEPLOYMENT_VARIABLES_NAME.APPLICATION_SLUG]: "NATS0",
    [DEPLOYMENT_VARIABLES_NAME.APPLICATION_VERSION]: "V2.1.9",
    [DEPLOYMENT_VARIABLES_NAME.APPLICATION_DESCRIPTION]:
      "Amazon Web Services provides Information Technology infrastructure services to businesses in the form of web services",
    [DEPLOYMENT_VARIABLES_NAME.APPLICATION_LOGO]:
      "https://d1q6f0aelx0por.cloudfront.net/product-logos/library-nats-logo.png",
  },
];

export const cloudVendor = { vendor: "aws", service: "eks" };

export const workflowData = [
  {
    name:
      "Process/cloudclapp-wf/Provision_Apps_EKS_Cluster/Process_Generate_Application_WF",
    displayName: "Restart APP",
    type: "UPDATE",
    visibility: 5,
    allowSchedule: null,
    tasks: null,
  },
  {
    name: "Process/cloudclapp-wf/Provision_Apps_EKS_Cluster/Process_Resume_APP",
    displayName: "Resume APP",
    type: "UPDATE",
    visibility: 5,
    allowSchedule: null,
    tasks: null,
  },
  {
    name: "Process/cloudclapp-wf/Provision_Apps_EKS_Cluster/Process_Pause_APP",
    displayName: "Pause APP",
    type: "UPDATE",
    visibility: 5,
    allowSchedule: null,
    tasks: null,
  },
];

export const userList = [
  {
    id: 130,
    baseRole: {
      id: 2,
      name: "Administrator",
    },
    externalReference: "NCLG130",
    login: "hto@openmsa.co",
    operator: {
      id: 33,
      abonnes: [],
      baseUrl: null,
      isPartner: false,
      name: "hto org 1103",
      prefix: "00u",
      address: {
        city: "",
        country: "",
        fax: "",
        mail: "",
        phone: "",
        streetName1: "",
        streetName2: "",
        streetName3: "",
        zipCode: "",
      },
      customersCount: 0,
      adminsCount: 0,
      managersCount: 0,
      externalReference: "hto org 1103",
    },
    acteurId: 495,
    address: {
      city: "",
      country: "",
      fax: null,
      mail: "hto@openmsa.co",
      phone: "",
      streetName1: "",
      streetName2: "",
      streetName3: "",
      zipCode: "",
    },
    delegationProfileId: 0,
    userType: 2,
    delegations: null,
    delegationsPerCustomer: {},
    netceloId: {
      ubiID: "00uG130",
      thePrefix: "00u",
      id: 130,
      customerPrefix: "00u",
    },
    isExternalAuth: false,
    activationKey: null,
    ccla: false,
    attachedCustomerIds: null,
    firstname: "",
    attachedOperatorIds: [33],
    manageAllUsers: false,
    name: "hto@openmsa.co",
    sort: "NAME",
    ldapAuthentication: false,
    delegationProfilePerCustomer: {},
  },
  {
    id: 125,
    baseRole: {
      id: 2,
      name: "Administrator",
    },
    externalReference: "NCLG125",
    login: "hto@openmsa.com",
    operator: {
      id: 29,
      abonnes: [],
      baseUrl: null,
      isPartner: false,
      name: "brand new hto org!",
      prefix: "00p",
      address: {
        city: "",
        country: "",
        fax: "",
        mail: "",
        phone: "",
        streetName1: "",
        streetName2: "",
        streetName3: "",
        zipCode: "",
      },
      customersCount: 0,
      adminsCount: 0,
      managersCount: 0,
      externalReference: "htoExtRef",
    },
    acteurId: 454,
    address: {
      city: "",
      country: "",
      fax: null,
      mail: "hto@openmsa.com",
      phone: "",
      streetName1: "",
      streetName2: "",
      streetName3: "",
      zipCode: "",
    },
    delegationProfileId: 0,
    userType: 2,
    delegations: null,
    delegationsPerCustomer: {},
    netceloId: {
      ubiID: "00pG125",
      thePrefix: "00p",
      id: 125,
      customerPrefix: "00p",
    },
    isExternalAuth: false,
    activationKey: null,
    ccla: false,
    attachedCustomerIds: null,
    firstname: "",
    attachedOperatorIds: [29],
    manageAllUsers: false,
    name: "hto@openmsa.com",
    sort: "NAME",
    ldapAuthentication: false,
    delegationProfilePerCustomer: {},
  },
];

export const monitoringGraph = [
  {
    id: "AVAILABILITY",
    name: "Availability",
    kpis: {
      latency: {
        color: "#24a36a",
        label: "Latency",
        id: "latency",
        suffix: "ms",
      },
      ttl: {
        color: "#0059a7",
        label: "TTL",
        id: "ttl",
      },
    },
    verticalLabel: "y",
    data: [
      {
        time: 1648485360000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648485600000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648485840000,
        latency: 0.25,
        ttl: 64,
      },
      {
        time: 1648486080000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648486320000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648486560000,
        latency: 2500,
        ttl: 64,
      },
      {
        time: 1648486800000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648487040000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648487280000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648487520000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648487760000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648488000000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648488240000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648488480000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648488720000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648488960000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648489200000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648489440000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648489680000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648489920000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648490160000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648490400000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648490640000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648490880000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648491120000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648491360000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648491600000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648491840000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648500240000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648500480000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648500720000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648500960000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648501200000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648501440000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648501680000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648501920000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648502160000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648502400000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648502640000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648502880000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648503120000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648503360000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648503600000,
        latency: 0.25,
        ttl: 64,
      },
      {
        time: 1648559520000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648559760000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648560000000,
        latency: 0,
        ttl: 64,
      },
      {
        time: 1648560240000,
        latency: 0.25,
        ttl: 64,
      },
      {
        time: 1648560480000,
        latency: 0,
        ttl: 64,
      },
    ],
  },
];
export const getEnvironmentResponse = [
  false,
  null,
  {
    envName: "hto env 01",
    envId: 144,
    envUbiqubeId: "00pA144",
    description: "2nd",
    cloudVendor: "Cloud/AWS",
    orgName: "htoExtRef",
    envIp: "127.0.0.1",
    envHostname: "temp-host",
    username: "12",
    envReference: "0419674e-a030-4fb9-b47a-94bcb8de6d0e",
    importFlag: false,
    serviceId: "126087",
    serviceName: null,
    envWFUri:
      "Process/cloudclapp-wf/Provision_an_EKS_Cluster__AWS_/Provision_an_EKS_Cluster__AWS_",
    appWFUri:
      "Process/cloudclapp-wf/Provision_Apps_EKS_Cluster/Provision_Apps_EKS_Cluster",
    deploymentCount: 49,
    userCount: 4,
    scanData: null,
    cclapOwner: 125,
    envEntityId: 323,
  },
  {
    "cache-control": "no-cache, no-store, max-age=0, must-revalidate",
    connection: "close",
    "content-length": "576",
    "content-type": "application/json",
    date: "Wed, 23 Mar 2022 04:05:00 GMT",
    expires: "0",
    pragma: "no-cache",
    server: "nginx/1.21.6",
    vary: "Accept-Encoding",
    "x-content-type-options": "nosniff",
    "x-frame-options": "DENY",
    "x-powered-by": "Express",
    "x-xss-protection": "1; mode=block",
    status: 200,
  },
  null,
];

export const environmentSummary = {
  environmentCount: 6,
  usersCount: 3,
  managerList: [
    {
      id: 142,
      baseRole: {
        id: 3,
        name: "Privileged Manager",
      },
      externalReference: "00pG142",
      login: "htoOrgPrivMan",
      operator: {
        id: 29,
        abonnes: [],
        baseUrl: null,
        isPartner: false,
        name: "brand new hto org!",
        prefix: "00p",
        address: {
          city: "",
          country: "",
          fax: "",
          mail: "",
          phone: "",
          streetName1: "",
          streetName2: "",
          streetName3: "",
          zipCode: "",
        },
        customersCount: 0,
        adminsCount: 0,
        managersCount: 0,
        externalReference: "htoExtRef",
      },
      acteurId: 515,
      address: {
        city: "",
        country: "",
        fax: null,
        mail: "hto@ubiqube.com",
        phone: "",
        streetName1: "",
        streetName2: "",
        streetName3: "",
        zipCode: "",
      },
      delegationProfileId: 0,
      userType: 2,
      delegations: null,
      delegationsPerCustomer: {},
      netceloId: {
        ubiID: "00pG142",
        thePrefix: "00p",
        id: 142,
        customerPrefix: "00p",
      },
      isExternalAuth: false,
      activationKey: null,
      ccla: false,
      attachedCustomerIds: [143, 144, 145, 151, 149, 150],
      firstname: "",
      attachedOperatorIds: null,
      manageAllUsers: true,
      name: "hto org priv man",
      sort: "NAME",
      ldapAuthentication: false,
      delegationProfilePerCustomer: {},
    },
    {
      id: 140,
      baseRole: {
        id: 4,
        name: "Manager",
      },
      externalReference: "00pG140",
      login: "htoOrgMan",
      operator: {
        id: 29,
        abonnes: [],
        baseUrl: null,
        isPartner: false,
        name: "brand new hto org!",
        prefix: "00p",
        address: {
          city: "",
          country: "",
          fax: "",
          mail: "",
          phone: "",
          streetName1: "",
          streetName2: "",
          streetName3: "",
          zipCode: "",
        },
        customersCount: 0,
        adminsCount: 0,
        managersCount: 0,
        externalReference: "htoExtRef",
      },
      acteurId: 511,
      address: {
        city: "",
        country: "",
        fax: null,
        mail: "hto@ubiqube.com",
        phone: "",
        streetName1: "",
        streetName2: "",
        streetName3: "",
        zipCode: "",
      },
      delegationProfileId: 0,
      userType: 2,
      delegations: null,
      delegationsPerCustomer: {},
      netceloId: {
        ubiID: "00pG140",
        thePrefix: "00p",
        id: 140,
        customerPrefix: "00p",
      },
      isExternalAuth: false,
      activationKey: null,
      ccla: false,
      attachedCustomerIds: [144],
      firstname: "",
      attachedOperatorIds: null,
      manageAllUsers: false,
      name: "hto org dev guy",
      sort: "NAME",
      ldapAuthentication: false,
      delegationProfilePerCustomer: {},
    },
    {
      id: 125,
      baseRole: {
        id: 2,
        name: "Administrator",
      },
      externalReference: "NCLG125",
      login: "hto@openmsa.com",
      operator: {
        id: 29,
        abonnes: [],
        baseUrl: null,
        isPartner: false,
        name: "brand new hto org!",
        prefix: "00p",
        address: {
          city: "",
          country: "",
          fax: "",
          mail: "",
          phone: "",
          streetName1: "",
          streetName2: "",
          streetName3: "",
          zipCode: "",
        },
        customersCount: 0,
        adminsCount: 0,
        managersCount: 0,
        externalReference: "htoExtRef",
      },
      acteurId: 454,
      address: {
        city: "",
        country: "",
        fax: null,
        mail: "hto@openmsa.com",
        phone: "",
        streetName1: "",
        streetName2: "",
        streetName3: "",
        zipCode: "",
      },
      delegationProfileId: 0,
      userType: 2,
      delegations: null,
      delegationsPerCustomer: {},
      netceloId: {
        ubiID: "00pG125",
        thePrefix: "00p",
        id: 125,
        customerPrefix: "00p",
      },
      isExternalAuth: false,
      activationKey: null,
      ccla: false,
      attachedCustomerIds: null,
      firstname: "",
      attachedOperatorIds: [29],
      manageAllUsers: false,
      name: "hto@openmsa.com",
      sort: "NAME",
      ldapAuthentication: false,
      delegationProfilePerCustomer: {},
    },
  ],
};


export const permissionProfiles = {
  environments: {
    general: {
      create: {
        name: 'create',
        access: false
      },
      tearDown: {
        name: 'tearDown',
        access: true
      },
      delete: {
        name: 'delete',
        access: false
      },
      access: true
    },

  },
  deployments: {
    general: {
      create: {
        name: 'create',
        access: true
      },
      action: {
        name: 'action',
        access: false
      },
      access: true
    },

  },
  applications: {
    dockerHub: {
      view: {
        name: 'view',
        access: true
      },
      access: true
    },
    virtualMachine: {
      view: {
        name: 'view',
        access: false
      },
      access: false
    },
    access: true
  },

  insights: {
    cost: {
      view: {
        name: 'view',
        access: true
      },
      access: true
    },
  },

  governance: {
    auditLogs: {
      view: {
        name: 'view',
        access: false
      },
      access: false
    },
  },
}
