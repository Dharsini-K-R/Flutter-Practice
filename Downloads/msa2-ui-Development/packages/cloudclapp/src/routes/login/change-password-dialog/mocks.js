export const linkValidationResponse = {
  message: "Link is valid",
};
