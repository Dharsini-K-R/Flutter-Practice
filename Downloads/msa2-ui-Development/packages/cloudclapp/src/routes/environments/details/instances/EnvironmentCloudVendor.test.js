import React from "react";
import { renderWithProvider, wait } from "cloudclapp/src/utils/test-utils";
import EnvironmentCloudVendor from "./EnvironmentCloudVendor";
import initialState from "cloudclapp/src/store/initialState";
import { cloudVendors } from "cloudclapp/src/routes/dashboard/mock";
import { environment } from "../mocks";

const mockState = {
  ...initialState,
  designations: { ...initialState.designations, cloudVendors },
};

describe("EnvironmentCloudVendor component", () => {
  beforeEach(() => {
    URL.createObjectURL = jest.fn();
  });
  it("renders correctly", async () => {
    const { container, getByText } = renderWithProvider({
      childComponent: <EnvironmentCloudVendor environment={environment} />,
      mockState,
    });
    await wait();
    expect(container.firstChild).toMatchSnapshot();
    expect(getByText("temp-host")).toBeTruthy();
    expect(getByText("127.0.0.1")).toBeTruthy();
  });
});
