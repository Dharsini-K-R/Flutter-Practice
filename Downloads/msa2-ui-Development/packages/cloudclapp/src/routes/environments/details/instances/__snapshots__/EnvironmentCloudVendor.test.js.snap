// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EnvironmentCloudVendor component renders correctly 1`] = `
<div
  class="MuiGrid-root MuiGrid-item"
>
  <div
    class="MuiBox-root MuiBox-root"
  >
    <div
      class="MuiBox-root MuiBox-root"
    >
      <i
        aria-hidden="true"
        class="la la-cloud makeStyles-icon makeStyles-default makeStyles-default"
        role="presentation"
      />
    </div>
    <div
      class="MuiBox-root MuiBox-root"
    >
      <p
        class="MuiTypography-root makeStyles-serviceDetails MuiTypography-body1"
        id="INSTANCE_DETAILS_ENVIRONMENT_SERVICE_NAME"
      >
        <span
          class="makeStyles-serviceName"
        >
          Elastic Container Service for Kubernetes
        </span>
         - 
        <span>
          AWS
        </span>
      </p>
      <p
        class="MuiTypography-root makeStyles-serviceDetails MuiTypography-body1"
        id="INSTANCE_DETAILS_ENVIRONMENT_DESCRIPTION"
      >
        aslkd adsg aslgd ll
      </p>
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-item"
      >
        <p
          class="MuiTypography-root makeStyles-serviceDetails MuiTypography-body1"
          id="INSTANCE_DETAILS_ENVIRONMENT_IP_ADDRESS"
        >
          <span
            class="makeStyles-boldText"
          >
            IP Address
            :
          </span>
           
          127.0.0.1
        </p>
        <p
          class="MuiTypography-root makeStyles-serviceDetails MuiTypography-body1"
          id="INSTANCE_DETAILS_ENVIRONMENT_HOSTNAME"
        >
          <span
            class="makeStyles-boldText"
          >
            Host Name
            :
          </span>
           
          temp-host
        </p>
      </div>
    </div>
  </div>
</div>
`;
